http/handler.go

```diff
@@ -115,7 +115,7 @@ func respondStandby(core *vault.Core, w http.ResponseWriter, reqURL *url.URL) {
 	redirectURL := url.URL{
 		Scheme:   advertiseURL.Scheme,
 		Host:     advertiseURL.Host,
-		Path:     reqURL.Path,
+		Path:     advertiseURL.Path + reqURL.Path,
```

Comment:
a) is `advertiseURL.Path` guaranteed to not to end with `/` ? Do we need to add path joining here ?
b) could you please push it upstream as well ?